fv3core/stencils/delnflux.py

```diff
@@ -131,9 +131,6 @@ def compute_delnflux_no_sg(
     else:
         # TODO: To join these stencils you need to overcompute, making the edges
         # 'wrong', but not actually used, separating now for comparison sanity.
-
-        # diffusive_damp(fx, fx2, fy, fy2, mass, damp, origin=diffuse_origin,
-        # domain=(grid.nic + 1, grid.njc + 1, nk))
```

Comment:
I think you were cleaning up and didn't need to delete this, but seems like it would minimally impact the delnflux refactor so is fine to leave.